app/models/weam.rb

```diff
@@ -10,7 +10,7 @@
 class Weam < ActiveRecord::Base
   include CsvHelper
 
-  REQUIRED_VET_TEC_LAW_CODE = 'vet tec only'
+  REQUIRED_VET_TEC_LAW_CODE = 'Educational Institution is Approved For Vet Tec Only'
```

Comment:
This needs to be lowercase

Reply:
Thanks, that was driving me nuts! Haven't had my caffeine yet. :)